src/pages/module-builder/Document.tsx

```diff
@@ -66,29 +73,56 @@ const Document = ({
                             let sectionPreview;
                             if (section.type === "staticContent") {
                                 sectionPreview = (
-                                    <MarkdownRenderer>
-                                        {section.markdown}
-                                    </MarkdownRenderer>
+                                    <Container
+                                        paddingTop={section.padding.top}
```

Comment:
Two things:
- We want to append "px" here
- Some basic validation here would be good: Try ``!Number.isNan(section.padding.top) ? `${section.padding.top}px` : "0px"``

Reply:
Will do.
The input is coerced to be a number in previous pr
![image](https://user-images.githubusercontent.com/24593519/150452927-77dad482-092e-4a5d-aa7f-2d1cef7848ff.png)